src/map/status.cpp

```diff
@@ -11445,8 +11445,6 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 				val1 = rnd()%ELE_ALL;
 			break;
 		case SC_CRITICALWOUND:
```

Comment:
Was this case supposed to be removed too? Right now it's going to fall into SC_MAGICMIRROR right below.

Reply:
Agree with aleos here. This looks incorrect.
If it should not decrease heal effectiveness, this case needs to be removed completely and also the references in the calculations in pc.cpp and skill.cpp need to be removed.

Reply:
Can you please change the code like you copied it above?
```
		case SC_CRITICALWOUND:
			// Level 1 ~ 5 & 6 ~ 10 has different duration
			// Level 6 ~ 10 use effect of level 1 ~ 5
			val1 = 1 + ((val1-1)%5);
			val2 = 20*val1;
			break;
```
This way it is more readable and additionally it fits our plans for the future better.
Thank you!

Reply:
Thanks for review. I ll do it soon when Im home